server/src/main/scala/actors/DrtSystemInterface.scala

```diff
@@ -377,13 +379,21 @@ trait DrtSystemInterface extends UserRoleProviderLike {
         GlaFeed(liveUrl, liveToken, livePassword, liveUsername, ProdGlaFeedRequester).tickingSource
       case "PIK" =>
         CiriumFeed(config.get[String]("feeds.cirium.host"), portCode).tickingSource(30 seconds)
+      case "EDI" =>
+        val currentDate = SDate.yyyyMmDdForZone(SDate.now(), DateTimeZone.UTC)
```

Comment:
This date needs to be dynamically calculated on each call to check the feed. If we leave it here it'll won't move forward as time passes.
I'd suggest passing the number of days ahead to start from, and the number of days to take, eg
live: 0, 2
forecast: 2, 180
And then calculate the date on each iteration when we call the end point

Reply:
Yeah , how can i do that :-(

Reply:
currentDate part of def liveArrivalsSource, so date would be dynamically calculated in each time liveArrivalsSource is def is called .  May I am thinking different . 
Is agree with 180 days though.